1618017466588 [handle_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Arm_moveit.py", line 99, in handle_command
    self.group.set_position_target([self.target.position.x,self.target.position.y,self.target.position.z],end_effector_link=end_link)
AttributeError: moveit_handle instance has no attribute 'target'
] 
1618017466612 [Finish initializing arm_moveit] 
